index.html

```diff
@@ -957,20 +957,37 @@ <h2>
         </p>
       </section>
     </section>
-    <section class="informative">
+    <section>
       <h2>
         Privacy and Security Considerations
       </h2>
-      <section>
-        <h3>
-          Access to aspects of a user's local computing environment
-        </h3>
-        <p>
-          The screen orientation type and angle of the device can be accessed
-          with the API specified in this document, and can be a potential
-          fingerprinting vector.
-        </p>
-      </section>
+      <p>
+        A screen's [=current orientation type|type=] and [=current orientation
+        angle|angle=] are a potential fingerprinting vector. The following
+        mitigation help protect a user's privacy by not revealing how a device
+        is being held, and also prevents the [=secondary=] orientation type and
+        associated angles from being user for fingerprinting purposes.
+      </p>
+      <p>
+        To resist fingerprinting (e.g., in private browsing), user agents MAY:
+      </p>
+      <ul>
+        <li>Depending on the screen's aspect ratio, only return either
+        {{OrientationType/"portrait-primary"}} or
+        {{OrientationType/"landscape-primary"}} for the value of the
+        {{ScreenOrientation/type}} attribute.
+        </li>
+        <li>For any [=natural=] orientation ([=portrait=] or [=landscape=]),
+        always return `0` for the value of the {{ScreenOrientation/angle}}
+        attribute. And for any rotated/opposite orientation, always return `90`
+        for the value of the {{ScreenOrientation/angle}} attribute.
```

Comment:
Okay, so this means we will be exposing the natural direction of the device. Or is the natural direction simply the direction the user used when navigating to the site?

Reply:
Oh! I quite like the idea of taking the orientation to which the page was navigated and using it to lie (i.e., and not expose the natural orientation). 

No browser does the above today, which is why I spec'ed it based on "natural" orientation, but that doesn't mean they couldn't.